### DISCUSSION TITLE
Allow new Maximum sample rate to be selectable

### DISCUSSION THREAD

**Discussion by dk0297** [fhdm-dev/scoppy#159] [Q&A]
After I set the Max Sample Rate to 2.0MS/s under RP2040 Settings it seems to only be available as a Auto Sample rate.
![Screenshot_20241202_204930_Scoppy](https://github.com/user-attachments/assets/e7615ad3-947b-4c81-bdfa-e6eeb7afe420)

Would it be possible to make it and 1.3MS/s selectable in the Sample rate dropdown if they are enabled in the RP2040 Settings?

**Comment by fhdm-dev** [comment]
Yep. Good idea. I forgot about this when I implemented the higher sample rates.
